repositories/permissions_requests.go

```diff
@@ -0,0 +1,121 @@
+package repositories
+
+import "github.com/topfreegames/Will.IAM/models"
+
+// PermissionsRequests repository
+type PermissionsRequests interface {
+	Clone() PermissionsRequests
+	Create(*models.PermissionRequest) error
+	Deny(string, string) error
+	Get(string) (*models.PermissionRequest, error)
+	Grant(string, string) error
+	ListOpenRequestsVisibleTo(*ListOptions, string) ([]models.PermissionRequest, error)
+	ListOpenRequestsVisibleToCount(string) (int64, error)
+	setStorage(*Storage)
+}
+
+type permissionsRequests struct {
+	*withStorage
+}
+
+func (prs *permissionsRequests) Clone() PermissionsRequests {
+	return NewPermissionsRequests(prs.storage.Clone())
+}
+
+func (prs *permissionsRequests) Create(pr *models.PermissionRequest) error {
+	_, err := prs.storage.PG.DB.Query(
+		pr, `INSERT INTO permissions_requests (service, ownership_level, action, resource_hierarchy,
```

Comment:
nit: break the query a bit more for improved readability? I think it particularly helpful to have each clause in its own line (INSERT / VALUES / WHERE / so on).

```golang
_, err := prs.storage.PG.DB.Query(
    pr,
    `INSERT INTO permissions_requests (
        service, ownership_level, action, resource_hierarchy, alias, message,
        state, service_account_id)
    VALUES (
        ?service, ?ownership_level, ?action, ?resource_hierarchy, ?alias,
        ?message, ?state, ?service_account_id)
    ON CONFLICT (
        service, ownership_level, action, resource_hierarchy, service_account_id)
      WHERE state = 'open'
      DO NOTHING
    RETURNING id`,
    pr,
)
```

Some people would do a parameter/column per line too, which can help with inserts as large as this one.

Same throughout.